app/views/bookmarks/show.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="row">
   <div class="col-md-9">
     <%= link_to @url, @url %>
-    <p><%= embed_description(@url) %></p>
+    <p><%= @bookmark.image_file_name %></p>
 
     <% if find_type(@url) == 'video' %>
       <%= embed_media(@url, 800) %>
```

Comment:
Both lines 6 and 7 trigger identical duplicate HTTP requests to the embedly API. Minimize number of external HTTP requests when rendering a page. 

External requests can greatly reduce the speed with which a user sees the final rendered page. If embedly's servers are unavailable, then our page will not be shown to the user.